tests/wpunit/Tribe/Tickets/Attendees_Test.php

```diff
@@ -248,6 +254,55 @@ public function test_event_details_top( Closure $fixture ): void {
 		}
 	}
 
+	/**
+	 * @test
+	 */
+	public function test_generate_filtered_list() {
+		$post_id = tribe_events()->set_args( [
+			'title'      => 'Test Event 1',
+			'status'     => 'publish',
+			'start_date' => '2020-01-01 09:00:00',
+			'end_date'   => '2020-01-01 11:30:00',
+		] )->create()->ID;
+
+		$rsvp_ticket_id = $this->create_rsvp_ticket( $post_id );
+
+		$this->create_attendee_for_ticket( $rsvp_ticket_id, $post_id, [
+			'full_name' => 'John Doe',
+		] );
+
+		$post_id = tribe_events()->set_args( [
+			'title'      => 'Test Event 2',
+			'status'     => 'publish',
+			'start_date' => '2020-01-02 09:00:00',
+			'end_date'   => '2020-01-02 11:30:00',
+		] )->create()->ID;
+
+		$rsvp_ticket_id = $this->create_rsvp_ticket( $post_id );
+
+		$this->create_attendee_for_ticket( $rsvp_ticket_id, $post_id, [
+			'full_name' => 'Jane Doe',
+		] );
+
+		$attendees = new Attendees();
+		$attendees->attendees_table = new Tribe__Tickets__Attendees_Table();
+
+		// Generate filtered list of attendees for the latter event.
+		$items = $attendees->generate_filtered_list( $post_id );
+
+		// Get the 'Ticket Holder Name' column from the arrays.
+		$this->assertEquals( $items[0][5], 'Ticket Holder Name' );
+		$this->assertEquals( $items[1][5], 'Jane Doe' );
```

Comment:
1) can we add a fail message to indicate what part of this test is failing?(here and elsewhere) - or split the tests up into logically named tests.
2) can we include a check that there is only one ticket in this iteration? i.e. `$this->assertCount( 2, $items, 'There should be only two tickets' )` (assuming that the count is tickets + header here)